back/app/uploaders/text_image_uploader.rb

```diff
@@ -1 +1,9 @@
-class TextImageUploader < BaseImageUploader; end
```

Comment:
This is a fix made by Alex for a bug with images in the Quill editor.

Reply:
@alexander-cit I think you can answer this better than me

Reply:
Yes, I think it's better to remove this change from this PR and do it as a separate task https://citizenlab.atlassian.net/jira/software/projects/TEC/boards/17?selectedIssue=TEC-109

Because we still need to test it properly.

Reply:
Removed this change again :+1: